pkg/logtailing/tailer.go

```diff
@@ -74,6 +74,17 @@ type EventPayload struct {
 	RequestID string `json:"request_id"`
 	Status    int    `json:"status"`
 	URL       string `json:"url"`
+	Error     string `json:"error"`
+}
+
+// RedactedError is the mapping for fields in error from an EventPayload
+type RedactedError struct {
```

Comment:
We can probably just call this `RequestLogError` or something. From the CLI's perspective it's not really important that the error is "redacted".